lib/os_places_api/client.rb

```diff
@@ -76,17 +76,38 @@ def any_locations?(results)
     end
 
     def build_locations(results)
+      return build_locations_from_ons(results) if ons_results_only?(results)
+
       locations = unfiltered_locations(results)
 
       raise NoResultsForPostcode if locations.empty?
 
       {
+        "source" => "Ordnance Survey",
+        "retired" => false,
+        "retired_on" => nil,
```

Comment:
Are we also adding a `retired_on` field, if so could we add to ADR?

Reply:
I'm going to remove this - I think retired_on isn't important enough for us to break out.